automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/billing/BillingManagementPage.scala

```diff
@@ -56,8 +56,8 @@ class BillingManagementPage(implicit webDriver: WebDriver) extends BaseFireCloud
     */
   def waitForCreateDone(projectName: String): Option[String] = {
     billingProjectTable.filter(projectName)
-    log.info(s"Waiting for new billing project $projectName to complete in 20 minutes with 10 seconds polling interval")
-    retry(10.seconds, 20.minutes)({
+    log.info(s"Waiting for new billing project $projectName to complete in 30 minutes with 30 seconds polling interval")
+    retry(30.seconds, 30.minutes)({
```

Comment:
start to seeing few failed builds because waiting for create didn't finish in 20 minutes. there were no errors. I'm increasing time out to 30 minutes.